src/content/docs/en/reference/modules/astro-actions.mdx

```diff
@@ -72,13 +76,21 @@ Return values are parsed using the [devalue library](https://github.com/Rich-Har
 **Type:** `ZodType | undefined`
 </p>
 
-The optional `input` property accepts a Zod validator (e.g. Zod object, Zod discriminated union) to validate handler inputs at runtime. If the action fails to validate, [a `BAD_REQUEST` error](#actionerror) is returned and the `handler` is not called.
+An optional property that accepts a Zod validator (e.g. Zod object, Zod discriminated union) to validate handler inputs at runtime. If the action fails to validate, [a `BAD_REQUEST` error](#actionerror) is returned and the `handler` is not called.
 
 If `input` is omitted, the `handler` will receive an input of type `unknown` for JSON requests and type `FormData` for form requests.
 
-##### Use with `accept: 'form'`
+#### `accept` property
 
-If your action accepts form inputs, use the `z.object()` validator to automatically parse form data to a typed object. All Zod validators are supported for form data fields (e.g. `z.coerce.date()`). Astro additionally provides special handling under the hood for your convenience for validating the following types of field inputs:
+<p>
+
+**Type:** `"form" | "json"`<br />
+**Default:** `json`
+</p>
+
+Defines the format expected by an action. If your action accepts form inputs, use `accept: "form"` with the `z.object()` validator to automatically parse `FormData` to a typed object.
+
+All Zod validators are supported for form data fields (e.g. `z.coerce.date()`). Astro additionally provides special handling under the hood for your convenience for validating the following types of field inputs:
```

Comment:
While we're here, could we document this as well? https://github.com/withastro/astro/issues/13562